### ISSUE TITLE
A11y and Numeronyms, draft proposal

### ISSUE THREAD

**Issue by scottaohara** [a11yproject/a11yproject.com#509]
Per the discussion going on over in [issue #504: Is the term "a11y" a barrier of inclusion and understanding?](https://github.com/a11yproject/a11yproject.com/issues/504), I thought I'd take a stab at writing [Quick Tip: A11y and a brief numeronyms primer](https://gist.github.com/scottaohara/3481723bb1790b14cfa2a460707f2e37).

Happy to get feedback / edit suggestions

Thanks

**Comment by nschonni** [comment]
Duplicate of #507

**Comment by svinkle** [comment]
@nschonni I don't see how this is a duplicate? #507 is a discussion on how to address numeronyms where this issue incorporates a draft proposal on said discussion. If anything I'd suggest merging the #507 thread here.

I'm planning to address the draft at some point. Curious if @jacobwarduk has any thoughts around the proposed article content?

**Comment by jacobwarduk** [comment]
Hi @svinkle, I think @scottaohara's done a good job of summarising the definition which would help, were the proposed path to be [G55: Linking to definitions](https://www.w3.org/TR/2016/NOTE-WCAG20-TECHS-20161007/G55).

However, I don't believe that the user having to navigate away from the page they are currently on, to find the definition to be particularly user friendly.

I would personally propose the recommended way to be [G97: Providing the first use of an abbreviation immediately before or after the expanded form](https://www.w3.org/TR/2016/NOTE-WCAG20-TECHS-20161007/G97), e.g.

 > When we talk about accessibility (A11Y), we are discussing...

 > The we has seen a massive increase in developers interested in A11Y (accessibility) and...

This not only allows the user to continue reading, but is used in context, rather than as an abstract concept they need to read about on another page.

Just my $0.02 :)

**Comment by svinkle** [comment]
@jacobwarduk agreed. Linking out to simply gaining an understanding for a definition would create unnecessary effort on the user's part.

Looking at your quote examples, I would expect the "a11y" numeronym to _not_ be used in formal writing, with the exception of this post where we explicitly define "a11y" for the reader. In any other instance, writing out the full word, "accessibility" should be the norm.

The other exception I can think of would be the hashtag "#a11y" -- this is common place to tag a Tweet, thought, or link, in order for the Tweet to have greater exposure. I'd imagine people general would know what this is, and if not, could click the hashtag to find out more.

Now I wonder about the current content set here; I should go through and update any instances of "a11y" where not appropriate.

Thanks for your $0.02! Very helpful. 🙂

**Comment by jacobwarduk** [comment]
@svinkle Agreed, my examples were not really appropriate, as discussion of the numeronym "A11Y" (accessibility) only really takes place in this context.

Given your current content set, it's not for me to say where you should or should not implement this. Although, I do believe that the [About](http://a11yproject.com/about.html) page might be a place to start with at least a brief definition.

**Comment by svinkle** [comment]
@scottaohara, taking another look at the article proposal, could add a few things (if not present already):

- Make sure to specify that the "a11y" numeronym is to not be used in formal writing. Rather, writing out the full word, "accessibility" should be the norm.
- If it is used for whatever context, the abbreviation/numeronym should be defined immediately before or after with the expanded form, e.g.
  > When we talk about accessibility (A11Y), we are discussing...

  > The number of developers interested in A11Y (accessibility) is rising quickly...
- Emphasize that some cases, such as the hashtag, "#a11y" or a band name, "A11yProject" are acceptable for greater post exposure and brand loyalty, respectively.

Other than this, I think it's a great post. 👍

**Comment by scottaohara** [comment]
thanks @svinkle i'll work on incorporating these ideas asap.

**Comment by svinkle** [comment]
And yes, @jacobwarduk, that's a good point about adding a quick note on the topic to the About page. Interested in writing something and sending a PR?

**Comment by jacobwarduk** [comment]
@svinkle Yep, I'll get something written up this evening.

**Comment by scottaohara** [comment]
Alright, @svinkle I've made some edits to incorporate the points you raised.

**Comment by svinkle** [comment]
@scottaohara Looking good! A few minor things to address:

- Revise this a bit -- too many instances of _"sure"_:
  > be sure to make sure people understand "accessibility = a11y". Sure, people should 
- Remove extra comma:
  > like Twitter, is a good thing!
- Start a new paragraph here as the article changes topics:
  > There are a lot of projects and sites out that that use "a11y" in their name.
- Make sue to use proper capitalization when referencing brand names ("The A11Y Project"):
  > Obviously there's the a11yproject

**Comment by scottaohara** [comment]
Edits made (which made me revise some more wording as well) and [submitted as a PR](https://github.com/a11yproject/a11yproject.com/pull/518)
